src/client/AddMealForm.js

```diff
@@ -0,0 +1,102 @@
+import React from "react";
+import { Header } from "./Header";
+import './AddMeal.css'
+
+const AddMealForm = ({
+  titleValue,
+  descValue,
+  dateValue,
+  locationValue,
+  onClick,
+  titleOnChange,
+  descOnChange,
+  locationOnChange,
+  dateOnChange,
+  maxReservationOnChange,
+  priceOnChange,
+  message
+}) => {
+
+    if (message.length !==0) {
+        return(
+            <>
+            <Header />
+            <p>{message}</p>
+            </>
+        )
+    } else {
+        return (
+            <>
+              <Header />
+              <form className="add-meal">
+                <div>
+                  <h3>Share your meal with others</h3>
+                </div>
+                <div>
+                  <label htmlFor="title">Meal title </label>
+                </div>
+                <div>
+                <input
+                    type={"text"}
+                    value={titleValue}
+                    onChange={titleOnChange}
+                    id="title"
```

Comment:
if you can add `name='title'` you can use `event.target.name` in your Handler function. If you can add `required` also it will detect automatically length of input and will warn user to fill this input. This is same for all inputs